Taxes/Operation/01_ManageDecisionMatrix.md

```diff
@@ -0,0 +1,256 @@
+
+The correct order of the decision-making criteria is crucial, as the system goes through the criteria in the order they are listed, starting from the top, and stops checking them as soon as a matching criterion is found. This means, that the criteria must be organized from specific to general to cover all possible relevant cases.
+
+
+## Create an entry
+
+Create an entry to define a new tax zone.
+
+#### Prerequisites  
+
+- An applicable tax class is created, see [Create a tax class](../Integration/02_ManageTaxClasses.md#create-a-tax-class).
+- An applicable tax zone is created, see [Create a tax zone](../Integration/03_ManageTaxZones.md#create-a-tax-zone).
+
+#### Procedures
+
+*Taxes > Settings > Tab DECISION MATRIX*
+
+![Decision matrix](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrix.png "[Decision matrix]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the upper left corner.   
+  A new data entry line is displayed.
+
+2. Click the *VAT ID* drop-down list and select the appropriate option. The following options are available:
+
+  - **Yes**: The end customer is a company and, therefore, has a VAT ID number (business-to-business transaction).
+  - **No**: The end customer is a private individual and, therefore, has no VAT ID number (business-to-customer transaction).
+  - **Any**: Wildcard; any value is accepted.
+
+
+3. Click the *Country of origin* drop-down list(s) and select the appropriate option(s). Use the drop-down list on the left to select a group of countries and the one on the right to select an individual country.
+
+  > [Info] The current version only provides the option **EU - European Union** as a group of countries. When a group of countries is selected, the drop-down list on the right is grayed out.
+
+4. Click the *Ship-to country* drop-down list(s) and select the appropriate option(s). Use the drop-down list on the left to select a group of countries and the one on the right to select an individual country.
+
+  > [Info] The current version only provides the option **EU - European Union** as a group of countries. When a group of countries is selected, the drop-down list on the right is grayed out.
+
+5. Click the *Ship-to postcode* drop-down list and select the appropriate option. The following options are available:
+
+  - **Range**: Enter a postcode range, for example, 27498-27498 to limit the criterion to a specific target area.
+  - **Regex**: Use regular expressions to specify postcodes containing alphanumeric characters, for example /^BT.*/.  
+
+  > [Info] There are many regular expressions testing tools available free of charge in the Internet, such as [Regex101](https://regex101.com/).
+
```

Comment:
Hier lässt du einen Schritt aus -> Nachdem ich in der Dropdown-liste range oder regex gewählt habe, wird das Feld rechts daneben freigeschaltet. 
Dann muss ich noch die range oder den regex in einem Schritt eingeben.